moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryMap.js

```diff
@@ -1,107 +1,80 @@
-    fontSize: "0.875rem",
-    fontWeight: 500,
-  },
-  toolTip: mapStyles.toolTipStyles,
-});
+const ProjectSummaryMap = ({ data }) => {
+  const [mapRef, mapRefState] = useMapRef();
+  const [basemapKey, setBasemapKey] = useState("streets");
 
-const ProjectSummaryMap = ({ projectFeatureCollection }) => {
-  const classes = useStyles();
-  const mapRef = useRef();
-  const featureCount = countFeatures(projectFeatureCollection);
+  const projectFeatureCollection = useMemo(() => {
+    const featureCollection = {
+      type: "FeatureCollection",
+      features: [],
+    };
 
-  /**
-   * Make use of a custom hook that returns a vector tile layer hover event handler
-   * and the details to place and populate a tooltip.
-   */
-  const { handleLayerHover, featureText, hoveredCoords } = useHoverLayer();
+    if (!data?.project_geography) return featureCollection;
 
-  const [viewport, setViewport] = useState(mapConfig.mapInit);
-  /**
-   * Make use of a custom hook that fits to a provided feature collection.
-   */
-  const { fitMapToFeatureCollectionOnRender } = useFeatureCollectionToFitBounds(
-    mapRef,
-    projectFeatureCollection,
-    true,
-    100
-  );
+    const projectGeographyGeoJSONFeatures = data.project_geography.map(
+      (feature) => makeFeatureFromProjectGeographyRecord(feature)
+    );
 
-  /**
-   * Updates viewport on zoom, scroll, and other events
-   * @param {Object} updatedViewPort - Mapbox object that stores properties of the map view
-   */
-  const handleViewportChange = useCallback(
-    (viewport) =>
-      setViewport((prevViewport) => ({ ...prevViewport, ...viewport })),
-    [setViewport]
-  );
+    return { ...featureCollection, features: projectGeographyGeoJSONFeatures };
+  }, [data]);
+
+  useZoomToExistingComponents(mapRefState, data);
 
-  /**
-   * Let's throw an error intentionally if there are no features for a project.
-   */
-  if (featureCount < 1) {
-    // Throw an error if this component was called without features to render.
-    throw Error("Map error: Cannot render or edit maps with no features");
-  }
+  const areThereComponentFeatures =
+    projectFeatureCollection.features.length > 0;
 
-  /**
-   * If we do have features, proceed to render map.
-   */
   return (
     <Box>
-      <Map
-        /* Current state of viewport */
-        {...viewport}
-        /* Object reference to this object */
-        ref={mapRef}
-        maxZoom={20}
-        style={{ width: "100%", height: "60vh" }}
-        /* Access Key */
-        mapboxAccessToken={MAPBOX_TOKEN}
-        onRender={fitMapToFeatureCollectionOnRender}
-        /* Get the IDs from the layerConfigs object to set as interactive in the summary map */
-        /* If specified: Pointer event callbacks will only query the features under the pointer of these layers.
-              The getCursor callback will receive isHovering: true when hover over features of these layers */
-        interactiveLayerIds={getSummaryMapInteractiveIds(
```

Comment:
We don't have any hover behavior scoped in this map or the component map yet so I'm removing the hover and interactive layer IDs needed to make layers hover-able.